streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/TopologyTest.scala

```diff
@@ -180,7 +191,7 @@ class TopologyTest extends JUnitSuite {
 
       // Compute the total per region by summing the individual click counts per region.
       val clicksPerRegion: KTableJ[String, JLong] = clicksByRegion
-        .groupByKey(Serialized.`with`(Serdes.String, Serdes.JavaLong))
+        .groupByKey(kstream.Serialized.`with`[String, JLong])
```

Comment:
Why need the `kstream.` prefix here?

Reply:
Wrong refactoring from my IDE. I will correct this.